packages/vm/test/tester/index.ts

```diff
@@ -45,7 +45,8 @@ import type { Common } from '@ethereumjs/common'
  * --profile                              If this flag is passed, the state/blockchain tests will profile
  */
 
-const argv = minimist(process.argv.slice(2))
+//@ts-expect-error Typescript thinks there isn't a default export on minimist but there is
+const argv = minimist.default(process.argv.slice(2))
```

Comment:
Side note: minimist is also causing problems with bun and we should consider to replace (should be simple).

Reply:
Okay, I think I might address this in a separate PR.  Not sure how much of a rewrite that will be to switch to yargs.

Reply:
Yeah, sure, definitely nothing for this PR and generally just a mention.